src/assignmentOverride.js

```diff
@@ -1,60 +1,44 @@
-var AssignmentOverride = (function() { // jshint ignore:line
-    var _AssignmentOverride = function(options) {
-        options = options || {};
-        this._visitor = options.visitor;
-        this._assignment = options.assignment;
-        this._username = options.username;
-        this._password = options.password;
+import $ from 'jquery';
+import TestTrackConfig from './testTrackConfig';
 
-        if (!this._visitor) {
-            throw new Error('must provide visitor');
-        } else if (!this._assignment) {
-            throw new Error('must provide assignment');
-        } else if (!this._username) {
-            throw new Error('must provide username');
-        } else if (!this._password) {
-            throw new Error('must provide password');
-        }
-    };
-
-    _AssignmentOverride.prototype.send = function() {
```

Comment:
this method is never used. i think it was copy/pasted from AssignmentNotification.